DigitalLearningSolutions.Web/Controllers/LogoutController.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace DigitalLearningSolutions.Web.Controllers
+{
+    using Microsoft.AspNetCore.Authentication;
+    using Microsoft.AspNetCore.Mvc;
+
+    public class LogoutController : Controller
+    {
+        [HttpGet]
+        public IActionResult Index()
+        {
+            HttpContext.SignOutAsync();
+            HttpContext.Response.Cookies.Delete("ASP.NET_SessionId");
```

Comment:
The old site uses an additional cookie – ASP.NET_SessionId – that is not used on the new site, but is generated if the user logs in on the new site, and then goes to the old site in the same session. If it is not deleted when logging out, then the user will still be logged in on the old site until they close the browser (ending the session). Deleting this cookie means that they will be immediately logged out on the old site when they log out on the new site (though they will still have to refresh the page to see the effect).